hips/draw/simple.py

```diff
@@ -172,6 +179,53 @@ def plot_mpl_hips_tile_grid(self) -> None:
         ax.imshow(self.image, origin='lower')
 
 
+class HipsDrawResult:
+    """Container class for reporting information related with fetching / drawing of HiPS tiles.
+
+    Parameters
+    ----------
+    image: `~numpy.ndarray`
+        Container for HiPS tile data
+    geometry : `~hips.utils.WCSGeometry`
+        An object of WCSGeometry
+    tile_format : {'fits', 'jpg', 'png'}
+        Format of HiPS tile
+    """
+
+    def __init__(self, image: np.ndarray, geometry: WCSGeometry, tile_format: str) -> None:
+        self.image = image
+        self.geometry = geometry
+        self.tile_format = tile_format
+
+    def __str__(self):
+        return (
+            'This class object contains two attributes: image and geometry'
+        )
+
+    def __repr__(self):
```

Comment:
Please follow the common pattern of mentioning the class name in the `repr`, like in this example:
https://github.com/hipspy/hips/blob/fce110c0e2bf31baba5d9050ccf08a9c67493dff/hips/tiles/tile.py#L99